functions.php

```diff
@@ -527,10 +532,75 @@ public function register_p4_page_type_taxonomy() {
 		}
 	}
 
+	/**
+	 * Save custom taxonomy for planet4 post types
+	 */
+	public function p4_save_post_type( $post_id ) {
+		// some of these checks might be redundant, but they're all nicely
+		// separated and easy to delete so I'll leave them for now.
+		// ignore autosave
+		if ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) {
+			return;
+		}
+		// check nonce
```

Comment:
Another minor tick, but inline comments should start with a capital letter and end with a stop character like a period.

Reply:
Added capitals and punctuation to single-line comments in this method.